src/flask/app.py

```diff
@@ -1332,7 +1332,11 @@ def decorator(f):
 
     @staticmethod
     def _get_exc_class_and_code(exc_class_or_code):
-        """Ensure that we register only exceptions as handler keys"""
+        """Ensure that we register only exceptions as handler keys
+
+        :param exc_class_or_code: the class for the exception, or
```

Comment:
"Any exception class, or an HTTP status code as an integer.

Reply:
Wrap at 72 characters with one indent level (I know the existing docs aren't consistent about this).